migrations/versions/1030_buyer_email_domain_data_migration.py

```diff
@@ -0,0 +1,47 @@
+"""Migrate buyer email domains from .txt file to DB
+
+Revision ID: 1030
+Revises: 1020
+Create Date: 2017-10-13 11:18:22.683693
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '1030'
+down_revision = '1020'
+
+buyer_domains_table = sa.Table(
+    'buyer_email_domains',
+    sa.MetaData(),
+    sa.Column('id', sa.Integer, primary_key=True),
+    sa.Column('domain_name', sa.String, nullable=True),
+)
+
+
+def upgrade():
+    try:
+        with open('./data/buyer-email-domains.txt') as f:
+            legacy_buyer_domains = [line.strip() for line in f]
+    except FileNotFoundError as e:
+        # When we eventually delete the file, we can exit the migration early
+        return
+
+    conn = op.get_bind()
+    for legacy_domain in legacy_buyer_domains:
+        # SELECT domain_name FROM buyer_email_domains WHERE domain_name == 'legacy-domain.org'
+        query = buyer_domains_table.select(
+            buyer_domains_table.c.domain_name == legacy_domain
+        ).with_only_columns([buyer_domains_table.c.domain_name])
```

Comment:
Don't think the `with_only_columns` gains us anything here. An `exists` would but only if we had tens of thousands of columns. Interesting though. Not a dealbreaker.

Reply:
Yep, good shout. I stuck `with_only_columns` in there as the last data migration (I think on the `Briefs` model) used it. Will remove so as to keep things as simple as possible.